README.rst

```diff
@@ -124,7 +124,7 @@ Want access to all the published data in bulk? If you're familiar with Python
 and `Jupyter Notebooks <https://jupyter.org/>`__ and are willing to install Docker you
 can:
 
-* `Download a PUDL data release <https://sandbox.zenodo.org/record/764696>`__ from
```

Comment:
Not sure why the sandbox was being referenced in the README... but I pointed this at the real data release concept DOI on the production server.